flixel/system/macros/FlxDefines.hx

```diff
@@ -90,12 +90,12 @@ class FlxDefines
 		abortMinVersion("Lime", "2.8.1", (macro null).pos);
 		#end
 
-		#if (openfl >= "4.0.0")
-		abortMaxVersion("OpenFL", "4.0.0", "3.6.1", (macro null).pos);
+		#if (openfl > "4.5.0")
+		abortMaxVersion("OpenFL", "4.5.1", "4.5.0", (macro null).pos);
```

Comment:
Hm, we definitely need to figure out what the issue with the latest versions is.